hoomd/mpcd/CollisionMethod.cc

```diff
@@ -57,6 +57,51 @@ void mpcd::CollisionMethod::collide(uint64_t timestep)
     rule(timestep);
     }
 
+void mpcd::CollisionMethod::checkCollisionWarnings(uint64_t timestep)
```

Comment:
I would prefer to call this `checkRigidBodyWarnings` given that is what it focuses on.

Reply:
Right now it does a check if any of the md particles are massless regardless of whether they're in a rigid body or a free particle, so the check isn't exclusive to rigid bodies. Even if the massless check should only be applied to rigid body particles, in the future there may be other scenarios that someone will want to add a warning for in this method that don't have to do with rigid bodies, and keeping the name a bit more general will account for that possibility. 

This method checks if there's anything about the state of the hoomd particles that would normally be bad when doing MPCD simulations. It just so happens that right now the things it checks for have to do with rigid bodies.

Reply:
OK, that’s fine then! However, just so you know, having an MD particle with no mass that isn’t in a rigid body probably will cause issues many other places in HOOMD before it gets here

Reply:
In that case, would you recommend checking for no mass only in rigid body particles or having separate warnings for massless rigid body particles and massless free particles?

Reply:
I think it's fine to just check them all at once (as you say, it's a problem for any massless particle to be coupled to the MPCD solvent). It was more of an FYI that the code is probably going to crash spectacularly with a divide by 0 error if there are any free particles with zero mass